### PR TITLE
PRL-3346 : Case status not changing to Withdrawn

### PR DESCRIPTION
**Pull request by sahaSonali**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java [hunk 1]
```diff
@@ -128,14 +125,16 @@ public class CallbackController {
...
-    private final AuthTokenGenerator authTokenGenerator;
     private final CourtFinderService courtLocatorService;
     private final LocationRefDataService locationRefDataService;
     private final UpdatePartyDetailsService updatePartyDetailsService;
     private final PaymentRequestService paymentRequestService;
     private final CourtSealFinderService courtSealFinderService;
     private final ConfidentialityTabService confidentialityTabService;
     private final LaunchDarklyClient launchDarklyClient;
+
+    @Qualifier("caseSummaryTab")
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java:136]: why do we need a qualifier?

File: config/owasp/suppressions.xml [hunk 2]
```diff
@@ -25,6 +25,7 @@
     <cve>CVE-2023-24998</cve>
     <cve>CVE-2023-28708</cve>
     <cve>CVE-2023-20861</cve>
+    <cve>CVE-2023-20860</cve>
```
Comment (AlokDatta) [inline config/owasp/suppressions.xml:28]: update with master

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java [hunk 3]
```diff
@@ -381,41 +378,21 @@ public AboutToStartOrSubmitCallbackResponse sendEmailNotificationOnCaseWithdraw(
...
-    private void sendWithdrawEmails(CaseData caseData, UserDetails userDetails, CaseDetails caseDetails) {
-        if (PrlAppsConstants.C100_CASE_TYPE.equalsIgnoreCase(caseData.getCaseTypeOfApplication())) {
-            solicitorEmailService.sendWithDrawEmailToSolicitorAfterIssuedState(caseDetails, userDetails);
-            Optional<List<Element<LocalCourtAdminEmail>>> localCourtAdmin = ofNullable(caseData.getLocalCourtAdmin());
-            if (localCourtAdmin.isPresent()) {
-                Optional<LocalCourtAdminEmail> localCourtAdminEmail = localCourtAdmin.get().stream().map(Element::getValue)
-                    .findFirst();
-                if (localCourtAdminEmail.isPresent()) {
-                    String email = localCourtAdminEmail.get().getEmail();
-                    caseWorkerEmailService.sendWithdrawApplicationEmailToLocalCourt(caseDetails, email);
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java:410]: are we missing our local court email notification?

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java (hunk 1 above)
Comment (sahaSonali) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/CallbackController.java:136]: We have @Qualifier present in CaseSummaryTabService, hence add the same here

**Review by AlokDatta — APPROVED** [review]

**Review by BharadwajSairam — APPROVED** [review]

**Pull request merged**